activities/PHQ-9-LORIS/items/phq9_0.jsonld

```diff
@@ -0,0 +1,17 @@
+{
+    "@context": [ "https://raw.githubusercontent.com/zaliqarosli/schema-standardization/2019-04-08-AddSubActivities/contexts/generic.jsonld",
+        "https://raw.githubusercontent.com/zaliqarosli/schema-standardization/2019-04-08-AddSubActivities/activities/PHQ-9-LORIS/phq9_context.jsonld"
+    ],
+    "@type": "https://raw.githubusercontent.com/ReproNim/schema-standardization/master/schemas/Field.jsonld",
+    "@id": "phq9_0",
+    "skos:prefLabel": "Header of PHQ-9 Questionnaire",
+    "skos:altLabel": "phq9_0",
+    "schema:description": "Header of the PHQ-9 Assessment",
+    "schema:schemaVersion": "0.0.1",
+    "schema:version": "0.0.1",
+    "question": "Questions of the PHQ-9 Questionnaire",
+    "ui": {
+        "inputType": null
```

Comment:
it might actually be useful to have `"inputType": "header"` instead of null and accompany it with `"headerLevel": (xsd:int)`. @akeshavan what do you think?

Reply:
I agree, null isn't very informative!